specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/monitors.json

```diff
@@ -0,0 +1,1190 @@
+            "schema": {
+              "$ref": "#/definitions/ProviderInstanceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/monitors/{monitorName}/providerInstances/{providerInstanceName}": {
+      "get": {
+        "tags": [
+          "ProviderInstances"
+        ],
+        "operationId": "ProviderInstances_Get",
+        "summary": "Gets properties of a provider instance.",
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "description": "Gets properties of a provider instance for the specified subscription, resource group, Monitor name, and resource name.",
+        "x-ms-examples": {
+          "Get properties of a SAP monitor Hana provider": {
+            "$ref": "./examples/workloadmonitor/ProviderInstances_Get.json"
+          },
+          "Get properties of a SAP monitor NetWeaver provider": {
+            "$ref": "./examples/workloadmonitor/NetWeaverProviderInstances_Get.json"
+          },
+          "Get properties of a OS provider": {
+            "$ref": "./examples/workloadmonitor/PrometheusOSProviderInstances_Get.json"
+          },
+          "Get properties of a Db2 provider": {
+            "$ref": "./examples/workloadmonitor/Db2ProviderInstances_Get.json"
+          },
+          "Get properties of a MsSqlServer provider": {
+            "$ref": "./examples/workloadmonitor/MsSqlServerProviderInstance_Get.json"
+          },
+          "Get properties of a PrometheusHaCluster provider": {
+            "$ref": "./examples/workloadmonitor/PrometheusHaClusterProviderInstances_Get.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/monitorNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderInstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ProviderInstance"
```

Comment:
>ProviderInstance [](http://example.com/codeflow?start=37&length=16)

Same here for secret

Reply:
Provider instance password field is marked as x-ms-secret and will not be present in any GET calls. We can verify that in the examples provided as well. Hence, resolving the conversation